### PR TITLE
C21 order new screen options are added

### PR DESCRIPTION
**Pull request by swaroopa86**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-3091

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java [hunk 1]
```diff
@@ -120,8 +120,7 @@ public AboutToStartOrSubmitCallbackResponse populateJudgeOrAdminDraftOrderCustom
         );
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
         DraftOrder selectedOrder = draftAnOrderService.getSelectedDraftOrderDetails(caseData);
-        if (selectedOrder != null && (CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(selectedOrder.getOrderType())
-            || CreateSelectOrderOptionsEnum.blankOrderOrDirectionsWithdraw.equals(selectedOrder.getOrderType()))
+        if (selectedOrder != null && (CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(selectedOrder.getOrderType()))
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java:123]: You dont need these many bracket.
How we are managing the existing features of withdraw order?

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/DraftAnOrderController.java [hunk 2]
```diff
@@ -89,8 +89,7 @@ public AboutToStartOrSubmitCallbackResponse populateFl404Fields(
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
         caseDataUpdated.put("caseTypeOfApplication", CaseUtils.getCaseTypeOfApplication(caseData));
 
-        if (!(CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(caseData.getCreateSelectOrderOptions())
-            || CreateSelectOrderOptionsEnum.blankOrderOrDirectionsWithdraw.equals(caseData.getCreateSelectOrderOptions()))
+        if (!(CreateSelectOrderOptionsEnum.blankOrderOrDirections.equals(caseData.getCreateSelectOrderOptions()))
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/DraftAnOrderController.java:92]: brackets and how are we catering the existing features of withdraw order?

File: src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CreateSelectOrderOptionsEnum.java [hunk 3]
```diff
@@ -18,43 +18,41 @@ public enum CreateSelectOrderOptionsEnum {
     directionOnIssue("directionOnIssue", "Direction on issue","2"),
     @JsonProperty("blankOrderOrDirections")
     blankOrderOrDirections("blankOrderOrDirections", "Blank order or directions (C21)","3"),
-    @JsonProperty("blankOrderOrDirectionsWithdraw")
-    blankOrderOrDirectionsWithdraw("blankOrderOrDirectionsWithdraw", "Blank order or directions (C21) - to withdraw application","4"),
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CreateSelectOrderOptionsEnum.java:22]: sure?

**Review by AlokDatta — COMMENTED** [review]
I understand we dont need C21 for withdraw anymore. But this ticket is about blank order for FL404B.

what about C21 withdraw for C100 cases?

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java (hunk 1 above)
Comment (AlokDatta) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java:123]: just brackets now. got the ticket https://tools.hmcts.net/jira/browse/PRL-3091

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/DraftAnOrderController.java (hunk 2 above)
Comment (AlokDatta) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/DraftAnOrderController.java:92]: just brackets now. got the ticket https://tools.hmcts.net/jira/browse/PRL-3091

**Comment by AlokDatta** [comment]
> I understand we dont need C21 for withdraw anymore. But this ticket is about blank order for FL404B.
> 
> what about C21 withdraw for C100 cases?

Can we get this feature tested in ITHC with merged code base please, after those minor comments addressed.

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java (hunk 1 above)
Comment (swaroopa86) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java:124]: Withdraw order features are available as part of new c21 order screen

**Pull request merged**